pkg/envoy/envoy.go

```diff
@@ -94,6 +94,21 @@ type Logger interface {
 	Log(entry *HttpLogEntry)
 }
 
+// LogEnvoyVersion logs envoy binary version
+func LogEnvoyVersion() {
+	out, err := exec.Command("cilium-envoy", "--version").Output()
+	if err != nil {
+		log.WithError(err).Fatal(`Envoy binary "cilium-envoy" cannot be executed`)
```

Comment:
Should we be more strict and only allow a particular version to be executed? (I've a feeling this is not a good idea for developers but might be a good idea for releases?)

Reply:
I was just thinking where a developer might be running an old and incompatible version of cilium-envoy.